src/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -34,14 +34,14 @@ public static void ThrownFromSerializerFails()
         {
             try
             {
-                JsonSerializer.Parse<IDictionary<string, int>>(@"{""Key"":1, ""Key"":2}");
-                Assert.True(false, "Expected JsonException was not thrown.");
+                JsonSerializer.Parse<IDictionary<string, string>>(@"{""Key"":1, ""Key"":2}");
+                Assert.True(false, "We follow 'Last value wins' approach for duplicate keys.");
```

Comment:
I don't think this message will be helpful in case such a test fails. Wouldn't a failure be due to a type mismatch (payload has string:int, we want string:string) and nothing to do with duplicate keys?